MediaStream.js

```diff
@@ -34,31 +34,60 @@ export default class MediaStream extends EventTarget(MEDIA_STREAM_EVENTS) {
    * unambiguously differentiate it from a local MediaStream instance not added
    * to an RTCPeerConnection.
    */
-  reactTag: string;
-
-  constructor(id, reactTag) {
-    super();
-    this.id = id;
-    // Local MediaStreams are created by WebRTCModule to have their id and
-    // reactTag equal because WebRTCModule follows the respective standard's
-    // recommendation for id generation i.e. uses UUID which is unique enough
-    // for the purposes of reactTag.
-    this.reactTag = (typeof reactTag === 'undefined') ? id : reactTag;
+  _reactTag: string;
+
+  constructor(arg) {
```

Comment:
can you describe the arguments please ?

Reply:
just by listing possible objects and where to find more info about each of them if they need explanation